src/OneinchCallee.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2023 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.6.12;
+pragma experimental ABIEncoderV2;
+
+interface GemJoinLike {
+    function dec() external view returns (uint256);
+
```

Comment:
This was because of the prettier that is enabled by default by [the vscode solidity extension](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity#formatting-using-prettier-and-the-prettier-solidity-plugin). Disabled it for now or do you use a standard linter/prettier config across all projects that I can find somewhere?

Reply:
Mmm we currently don't use a linter afaik, so it's mostly manual code-style work/reviewing. I personally don't use vscode atm.

Reply:
Do you at least have those rules described in a natural language somewhere? In other words, how can we get and overview of all rules and pass this knowledge to other team members? 🙂 

> I personally don't use vscode atm

In principle, you don't have to use specific code editor to lint or prettify the code, most of the time those are standalone libraries, e.g.: https://consensys.github.io/smart-contract-best-practices/security-tools/linters-and-formatters/